src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -58,7 +58,7 @@ String courseString(Set<Course> courseList) {
     String tutorialString(Set<Tutorial> tutorialList) {
```

Comment:
can name methods as getTutorialString(...) instead?
can help do that for the other methods in ListCommand also